tests/test_pyjsonviewer.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
```

Comment:
Could you please add a test for `sample.json`?

Reply:
I copied `sample.json`  into `dictionary.json` so `test_dictionary_file` covers `sample.json`.

Reply:
Ah. Make sense. So, can you merge dictionary.json and list.json. I think list.json can be a part of a json element of dictionary.json. Is it impossible?

Reply:
You are right. I can add another test file `merged.json` which will have list as a field in dictionary object. Then I can add corresponding test case for those kind of files.

I hope that is what you are suggesting.

Reply:
Thanks. I want to keep a test json file is a one file, `sample.json`. I want to put all kinds of supports json representation into a single json file. That helps to explain this tool functionalities.

Reply:
Actually code was already working for nested objects. `sample.json` has a fields `GlossSeeAlso` etc which are lists.
But it wasnt working for json files _which were lists to begin with._ 
That"s why I had to add seperate files `dictionary.json` which was an object and `list.json` which is a list.

By keeping them in same file we wont be able to test these cases seperately.

Reply:
> But it wasnt working for json files which were lists to begin with.
> That"s why I had to add seperate files dictionary.json which was an object and list.json which is a list.

Oh. Make sense. I understood why you are using two files.